### ISSUE TITLE
About pop up

### ISSUE THREAD

**Issue by ARoddaTyler** [qorf/quorum-studio#3]
A screen reader can navigate to the "about" menu, but the pop up is not accessible.

**Comment by andreas-stefik** [comment]
Apparently, the about dialog was broken in Quorum Studio in a number of annoying ways. This is now fixed.
